test/test_image.py

```diff
@@ -117,20 +122,19 @@ def test_write_jpeg(self):
 
             basedir = os.path.dirname(img_path)
             filename, _ = os.path.splitext(os.path.basename(img_path))
-            torch_jpeg = os.path.join(
-                basedir, '{0}_torch.jpg'.format(filename))
-            pil_jpeg = os.path.join(
+            torch_jpeg_path = os.path.join(
+                TMP_DIR, '{0}_torch.jpg'.format(filename))
+            pil_jpeg_path = os.path.join(
                 basedir, 'jpeg_write', '{0}_pil.jpg'.format(filename))
 
-            write_jpeg(img, torch_jpeg, quality=75)
+            write_jpeg(img, torch_jpeg_path, quality=75)
```

Comment:
I wonder if we could even get rid of the `jpeg_write` folder completely. 

Unfortunately I don't fully understand the reasons why this was introduced. Perhaps @andfoy could give us some info on whether he thinks this is needed.

Reply:
This is done in order to prevent the reading test function from taking those assets up

Reply:
@andfoy I was referring to the fact that we maintain a separate pil version of the image inside the assets and we keep it in version control. I wonder if it's OK to start writing to a temporary folder and remove also the _pil image version from the assets. Thoughts?

Reply:
That's a separate topic, we keep those files because we cannot compare directly the output between io functions and PIL due to the difference between libjpeg and libjpeg-turbo

Reply:
I see. Thanks for the clarification. Then we can leave this file in the assets.